hack/runner-release-watch.sh

```
@@ -0,0 +1,269 @@
+    return
+  fi
+  esc=${latest//./\\.}
+  # The version-anchored grep already drops Windows tags, whose OS token comes
+  # after the version; the filter is belt and braces if that shape changes.
+  flavours=$(printf '%s' "${json}" | jq -r '.results[].name' \
+    | grep -vE 'nanoserver|servercore' | sed -e 's/-pwsh$//' \
+    | grep -E -- "(^|-)v${esc}\$" | sed -E "s/-?v${esc}\$//" \
+    | sed -E 's/-?(x86_64|arm64|arm|ppc64le|riscv64|s390x)$//' \
+    | sed -e 's/^$/(default)/' | sort -u)
+  printf 'Helper image published for v%s (%s tags). Linux flavours:\n\n' "${latest}" "${total}"
+  printf '%s\n' "${flavours}" | sed 's/^/- `/;s/$/`/'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)